docs/01-startgids.md

```diff
@@ -0,0 +1,98 @@
+In de toolkit is een <a>digitoegankelijkheid sectie (onder ontwikkeling)</a> ingericht waar uitleg en hulp wordt gegeven om te helpen bij het voldoen aan de digitoegankelijkheid. 
+
+</br>
+# Waar kan de toolkit mij bij helpen? 
+Per teamrol is beschreven welke secties interessant zijn en alle tools en middelen die handig zijn om te gebruiken om dubbel werk te voorkomen.
+
+## Productowner
+In het kader van tijdsbesparing laat de toolkit zien wat al beschikbaar is zoals templates, componenten maar ook inzicht in veel voorkomende fouten. De sectie betreffende digitoegankelijkheid is ook interessant. Om de digitoegankelijkheid succesvol toe te passen moet een heel team zich verantwoordelijk voelen en bijdragen aan het controleren van de diverse aspecten die digitoegankelijkheid met zich meebrengt.  
+Het omgevingsloket wordt ontwikkeld vanuit het gebruikersperspectief. Daarom zijn de persona’s  ook te vinden in de toolkit.  
+
+Neem de tijd om het platform door te nemen. Om vervolgens je team te briefen over welke secties mogelijk handig en interessant zijn voor de verschillende specialisaties binnen het team.  
+Belangrijk is dat ze weten waar alle beschikbare bestanden te vinden zijn.
+
+Links naar relevante pagina's
+</br>
+<a href="">Digitoegankelijkheid(onder ontwikkeling)</a>
+</br>
+<a href="">Sketch library</a>
+</br>
+<a href="https://github.com/dso-toolkit/dso-toolkit/blob/master/README.md" target="_blank">Installation guide(English)</a>
+</br>
+<a href="/components/examples/">Templates</a>
+</br>
+<a href="">Meewerken aan de toolkit(onder ontwikkeling)</a>
+
+
+## Ontwerper
+Om te beginnen is er een <a>sketch library</a> waar alle componenten staan die ook in de toolkit staan.Hoe deze componenten moeten worden toegepast is terug te vinden in de toolkit. Onzeker of het juiste component in de juiste context gebruikt wordt? Raadpleeg dan de toolkit.  
+Hier zijn ook <a>templates</a> te vinden voor common practice pagina structurering en indeling. Ook is het belangrijk om de pagina betreffende <a>digitoegankelijkheid</a> door te nemen.  
+Als ontwerper kun je er voor kiezen om pagina’s direct te ontwerpen met de toolkit, doormiddel van het handlebar.js kunnen templates snel gemaakt worden.  
+Probeer te voorkomen dat er onderdelen gemaakt worden die al gerealiseerd zijn. Ontkom je er toch niet aan om een nieuw component te maken dat toch belangrijk is voor de toolkit. Bekijk dan de <a>proces pagina</a> betreffende de toolkit. Hoe je nieuwe componenten of updates op bestaande componenten kan doorvoeren.
+
+Links naar relevante pagina's
+</br>
+<a href="">Digitoegankelijkheid(onder ontwikkeling)</a>
+</br>
+<a href="">Sketch library</a>
+</br>
+<a href="/docs/colors/">Kleurgebruik</a>
```

Comment:
Deze URLs gaan niet werken op de productie omgeving. De root dir is het versienummer. Ik ga hier naar kijken